src/main/java/com/ixortalk/authserver/config/IxorTalkProperties.java

```diff
@@ -204,4 +214,26 @@ public void setUrl(String url) {
             this.url = url;
         }
     }
+
+    public static class Logout {
+
+        private String defaultRedirectUri;
```

Comment:
Default for this redirect URI should be set to what it was before (hard-coded), otherwise all existing instances will be broken.